.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
Hmm, I think [we](https://github.com/codeforboston/cliff-effects/pull/824#issuecomment-426595109) wanted to specify an actual version number. Would that be acceptable?